<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageToSTL</class>
 <widget class="QMainWindow" name="ImageToSTL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageToSTL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="title">
         <string>Information</string>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>181</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Cursor position(x,y,z):</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelXCoordinate">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelZCoordinate">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>50</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="labelYCoordinate">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>50</y>
           <width>41</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>191</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Intensity under cursor:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Layer&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelImageName">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>94</x>
           <y>100</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Intensity&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelImageIntensity">
         <property name="geometry">
          <rect>
           <x>94</x>
           <y>120</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>160</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Level:</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>185</y>
           <width>60</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Window:</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="lineEditSourceImageLevel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>160</y>
           <width>85</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimum">
          <double>-66000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>66000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>10.000000000000000</double>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="lineEditSourceImageWindow">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>185</y>
           <width>84</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimum">
          <double>-66000.000000000000000</double>
         </property>
         <property name="maximum">
          <double>66000.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>10.000000000000000</double>
         </property>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QScrollBar" name="sliceSourceImagePosition">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Slice selection</string>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QVTKWidget" name="qvtkWidgetDisplaySourceImage"/>
      </item>
      <item row="0" column="2">
       <widget class="QVTKWidget" name="qvtkWidgetPolyData"/>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="2">
         <widget class="QLabel" name="divideLineLabel1">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;of&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelSourceImageSliceNum">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="labelSourceImageSliceTotal">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="separator"/>
    <addaction name="actionSavePly"/>
    <addaction name="actionSaveSTL"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuParameterSet">
    <property name="title">
     <string>ParameterSet</string>
    </property>
    <addaction name="actionSTLSet"/>
   </widget>
   <widget class="QMenu" name="menuScreenshot">
    <property name="title">
     <string>Screenshot</string>
    </property>
    <addaction name="actionScreenshotLeft"/>
    <addaction name="actionScreenshotRight"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionDisplayPly"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuParameterSet"/>
   <addaction name="menuScreenshot"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenImage">
   <property name="text">
    <string>OpenImage</string>
   </property>
  </action>
  <action name="actionSaveSTL">
   <property name="text">
    <string>SaveSTL</string>
   </property>
  </action>
  <action name="actionSTLSet">
   <property name="text">
    <string>STLSet</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionScreenshotLeft">
   <property name="text">
    <string>ScreenshotLeft</string>
   </property>
  </action>
  <action name="actionScreenshotRight">
   <property name="text">
    <string>ScreenshotRight</string>
   </property>
  </action>
  <action name="actionDisplayPly">
   <property name="text">
    <string>DisplayPly</string>
   </property>
  </action>
  <action name="actionSavePly">
   <property name="text">
    <string>SavePly</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ImageToSTL.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>ImageToSTL</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpenImage</sender>
   <signal>triggered()</signal>
   <receiver>ImageToSTL</receiver>
   <slot>openSourceImageSlot()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliceSourceImagePosition</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImageToSTL</receiver>
   <slot>setSourceImageSliceSlot(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>709</x>
     <y>345</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditSourceImageLevel</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ImageToSTL</receiver>
   <slot>setSourceImageLevel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditSourceImageWindow</sender>
   <signal>valueChanged(double)</signal>
   <receiver>ImageToSTL</receiver>
   <slot>setSourceImageWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openSourceImageSlot()</slot>
  <slot>setSourceImageSliceSlot(int)</slot>
  <slot>setSourceImageLevel()</slot>
  <slot>setSourceImageWindow()</slot>
 </slots>
</ui>
